app/models/user.rb

```diff
@@ -1,4 +1,6 @@
 class User < ApplicationRecord
+  devise :database_authenticatable, :registerable, :recoverable, :rememberable, :validatable
+
```

Comment:
- [ ] Great work setting up `devise` for user authentication. Kindly activate the `:confirmable` attribute of `devise` for users in this application as it is a requirement. The `:confirmable` module ensures the application asks for confirmation of email from users after signing up. 👍 😎